frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/DetaljerTiltaksgjennomforingerPage.tsx

```diff
@@ -55,41 +53,28 @@ export function DetaljerTiltaksgjennomforingerPage() {
         </div>
       </Header>
 
-      <Tabs
-        value={tabValgt}
-        onChange={(value) => setTabValgt(value as TiltaksgjennomforingerTabs)}
-      >
+      <Tabs value={currentTab()} >
```

Comment:
Dette ser bittelitt repetativt ut (if-statement for å finne tab, samt utlistingen av tabs under) når jeg leser koden, men jeg klarer ikke helt å se om det er en aktuell kanditat for en generell React-komponent eller ikke.

Tenker noe ala. følgende, der RoutableTabs tar seg av utlede currentTab samt rendring av NavLink's osv, men kan hende oppførselen er forskjellig på tvers av sidene som gjør at generaliseringen ikke passer helt inn?

```tsx
    const tabs = [
        {
            value: "detaljer",
            label: "Detaljer"
        },
        {
            value: "notater",
            label: "Notater"
        },
        {
            enabled: visDeltakereFraKometFeature,
            value: "deltakere",
            label: "Deltakerliste"
        },
    ]
    
    <RoutableTabs base={`/tiltaksgjennomforinger/${tiltaksgjennomforing.id}`} tabs={tabs} />
```

Reply:
Hmm kanskje. Etter jeg fiksa UU forsvant NavLinks. Men så måtte jeg sette `aria-controls` manuelt på tabsene og de er ganske custom for hver tab. Tror det er bedre å la det være sånn det er.